index.html

```diff
@@ -0,0 +1,49 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />    <link rel="stylesheet" href="styles.css">
+    <script src="./script.js"></script>
+  </head>
+  <body>
+    <main>
+      <header class="flex">
+        <h2>My Logo</h2>
+        <div class="flex icon_hamburger__container">
+          <span id="icon-hamburger"></span>
+        </div>
+      </header>
+      <section class="headline">
+        <h1>I'm Mariane <br> Glad to see <br> you!</h1>
+        <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <div class="flex headline__socials">
+          <h6>LET'S CONNECT</h6>
+          <div class="icons_social__container flex">
+            <i class="fa-brands fa-twitter"></i>
+            <i class="fa-brands fa-linkedin-in"></i>
+            <i class="fa-brands fa-github"></i>
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-instagram"></i>
+          </div>
+        </div>
+        <div class="background_shapes__container">
+          <span id="top-circle-edge" class="icon-circle-edge"></span>
+          <span id="top-small-rectangle" class="icon-small-rectangle"></span>
+          <span id="top-small-circle" class="icon-small-circle"></span>
+          <span id="top-right-big-rectangle" class="icon-big-rectangle"></span>
+          <span id="top-right-big-ellipse" class="icon-big-half-ellipse"></span>
+          <span id="top-left-half-circle" class="icon-half-circle"></span>
+          <span id="top-left-medium-rectangle" class="icon-medium-rectangle"></span>
+          <span id="middle-right-circle-edge" class="icon-circle-edge"></span>
+          <span id="bottom-circle" class="icon-circle"></span>
+          <span id="bottom-medium-rectangle" class="icon-medium-rectangle"></span>
+          <span id="bottom-right-half-circle" class="icon-half-circle"></span>
+          <span id="bottom-right-small-ellipse" class="icon-small-half-ellipse"></span>
+          <span id="bottom-left-circle-edge" class="icon-circle-edge"></span>
```

Comment:
Kindly make sure you download the entire background image in Figma. those little images are not supposed to be downloaded separately. As you can notice they make your mark up file looks ugly and as a result a pretty long Style file.